src/nidmm/metadata/functions.py

```diff
@@ -1083,6 +1085,14 @@
             'description': '\nReturns the error information associated with the\n**Instrument_Handle**. This function retrieves and then clears the\nerror information for the session. If you leave the\n**Instrument_Handle** unwired, this function retrieves and then clears\nthe error information for the process.\n'
         },
         'is_error_handling': True,
+        'method_templates': [
```

Comment:
These metadata files are supposed to be copied exactly from internal NI artifacts. It appears you edited them here in source?

What's the plan?

Reply:
He has an open thread that says

>FYI, I'll copy the metadata changes to the original sources after this PR is accepted.

Reply:
Ah thanks! Forgot. After accepted but before merged please 😉

Reply:
@marcoskirsch were you asking him to copy internal metadata here or to copy these metadata changes to our internal source?

Reply:
I'm not done yet. Just testing the updates. I'm about to post a PR for the internal source changes, and once we get exports from those changes, I'll copy it back here again.

Reply:
https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/366375

Reply:
I was asking

>  to copy internal metadata here

The metadata on GitHub shouldn't be hand-edited.

Reply:
Done